addons/mediarecorder/style.css

```diff
@@ -34,11 +38,19 @@ p.mediaRecorderPopupOption {
 
 .mediaRecorderPopupOption input[type="checkbox"] {
   height: 1.5rem;
+  width: 15px;
```

Comment:
I think it's better to keep the browser's default checkbox size for consistency with other popups that contain checkboxes (Make a Block, gamepad settings).